notebooks/eda/EDA.md

```diff
@@ -0,0 +1,131 @@
+# Exploratory Data Analysis (EDA) Report
```

Comment:
This PR seems to be incomplete as it does not have the corresponding notebook explaining the EDA done for this project as mentioned in this issue https://github.com/slalom-ubc-mds/Power-Price-Prediction/issues/20. For now, we've added selective EDA in the final report. Please raise this PR with a notebook explaining the key EDA of our project utilizing this script.